ww3tools/ProcSat_Altimeter.py

```diff
@@ -146,8 +80,13 @@ def along_track(AODN,atime,wconfig):
                     shscal = np.append(shscal,np.nanmean(AODN['HS_CAL'][indt[indseg]]))
                     swsp = np.append(swsp,np.nanmean(AODN['WSPD'][indt[indseg]]))
                     swspcal = np.append(swspcal,np.nanmean(AODN['WSPD_CAL'][indt[indseg]]))
+
                     # print(repr(t)+" "+repr(i))
                     i=i+1
+    swdepth = np.array(swdepth)
```

Comment:
Is it okay to do this for every satelite or should there be a "if JASON2" statement here?

Reply:
The code was tested with other satellites like JASON3, CRYOSAT2, and others on Hera, and it worked without any problems.

Reply:
Thank you, I will now independently do a test and will report back.